R/style.R

```diff
@@ -0,0 +1,136 @@
+#' Specify styling for Sphere
+#'
+#' Returns a list for the \code{style = list(sphere = list())} function,
+#' for setting the sphere specifications.
+#'
+#'@param hidden Boolean - do not show atom. Default \code{FALSE}
+#'@param radius Override van der waals radius.
+#'@param scale Scale radius by specified amount.
+#'@param colorscheme Specify scheme to color the atoms by. Default is "default".
+#'Other choies are "Carbon", ssPyMOL", "ssJmol", "Jmol", "default", "amino",
+#'"shapely", "nucleic", "chain", "chainHetatm", "prop".
+#'@param color Discrete, fixed coloring, overrides any colorscheme.
+#'@param opacity Opacity of spheres, 0 being invisible. Must be the same for
+#'all atoms in the model.
+#'
+#' @return R3dmol \code{id} or a \code{r3dmol} object (the output from
+#' \code{r3dmol()})
+#' @export
+#'
+
+
+m_sphere_style <- function(hidden=F,
```

Comment:
[should] Use `FALSE` instead of `F`, and space should be add in ( `hidden = FALSE` )
Tips: Before commits, it's better to use `styler::style_pkg()` to make the codes has been formatted.